FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -194,22 +197,44 @@ extension ComposeViewController {
     }
 
     @objc private func handleSendTap() {
-        sendMessage()
+        prepareSigningKey()
     }
 }
 
 // MARK: - Message Sending
 
 extension ComposeViewController {
-    private func sendMessage() {
+    private func prepareSigningKey() {
+        guard let key = try? keyService.getPrvKeyInfo().get().first else {
+            showAlert(message: "No available private key has your user id \"\(email)\" in it. Please import the appropriate private key.")
+            return
+        }
+
+        guard key.passphrase != nil else {
+            let alert = AlertsFactory.makePassPhraseAlert(
```

Comment:
Does `makePassPhraseAlert` also check that the pass phrase is correct for the key? (I don't have the code in front of me)